components/has/.tekton/pvc.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: app-studio-default-workspace
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
```

Comment:
How do we want control the size of the workspace? The default 1GB can be sufficient for many projects, but some will require much more space, especially if there are a few runs left on the workspace.

Reply:
Good point @goldmann .
We could come up with a guidance ( which we'll need anyway for our users ) - but at the moment, I would leave it to the component owners to make the right call.